pkg/ensurance/collector/nodelocal/disk.go

```diff
@@ -0,0 +1,123 @@
+package nodelocal
+
+import (
+	"io/ioutil"
+	"time"
+
+	"github.com/shirou/gopsutil/disk"
+	"k8s.io/klog/v2"
+
+	"github.com/gocrane/crane/pkg/common"
+	"github.com/gocrane/crane/pkg/ensurance/collector/types"
+)
+
+const (
+	diskioCollectorName = "diskio"
+	sysBlockPath        = "/sys/block"
```

Comment:
does this fit for all systems? or for linux only?

Reply:
It only supports linux, not support windows and mac.  I add an building flag in the file.